azuredeploy.bicep

```diff
@@ -327,6 +985,230 @@ resource serverName_databaseName 'Microsoft.DBforMySQL/flexibleServers/databases
   }
 }
 
+// Azure Virtual Desktop and Session Hosts region
+
+resource hostPool 'Microsoft.DesktopVirtualization/hostPools@2022-10-14-preview' = {
```

Comment:
I think this API version does not support getting the registration token. See, for example, this discussion.